client/src/components/Sidebar/menu/SubMenuCloseButton.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { createPortal } from 'react-dom';
+
+import Icon from '../../Icon/Icon';
+import { gettext } from '../../../utils/gettext';
+
+export default function SubMenuCloseButton({isVisible, dispatch}) {
+  if (!isVisible) {
+    return null;
+  }
+  return createPortal(
```

Comment:
Another issue with nested submenus is that this will create multiple buttons in the DOM. But again, that's not something we officially support for now, so I think we can handle that later.